web/index.ts

```diff
@@ -1,7 +1,31 @@
-import { load, Root } from 'protobufjs';
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {load, Root} from 'protobufjs';
+const protoJSON = require('./messages.json');
 
 function loadProtocolBuffers(): Promise<Root> {
+  if (typeof window !== 'undefined') {
+    return Promise.resolve(Root.fromJSON(protoJSON));
+  }
+
   return load(`${__dirname}/../proto/messages.proto`);
```

Comment:
I feel there is an unneeded branching here. 
Since the `messages.json` is derived from `messages.proto` maybe we can return the `Root.fromJSON(protoJSON)` for both node and the browser (and remove the `if` condition and line 28